src/main/kotlin/no/nav/tilleggsstonader/sak/interntVedtak/InterntVedtakService.kt

```diff
@@ -162,6 +165,27 @@ class InterntVedtakService(
         }
     }
 
+    private fun mapVedtaksperioder(
+        vedtak: Vedtak?,
+        behandling: Saksbehandling,
+    ): List<Vedtaksperiode> {
+        val vedtaksperioder =
+            vedtak
+                ?.takeIfType<InnvilgelseTilsynBarn>()
+                ?.data
+                ?.beregningsresultat
+                ?.tilDto(behandling.revurderFra)
+                ?.vedtaksperioder
+        return vedtaksperioder?.map {
+            Vedtaksperiode(
+                målgruppe = it.målgruppe,
+                aktivitet = it.aktivitet,
+                fom = it.fom,
+                tom = it.tom,
+            )
+        } ?: emptyList()
```

Comment:
Burde denne endringen håndtere både tilsyn barn og læremidler? 
For læremidler vil man nå vise tittel "Vedtaksperioder" og en tom liste for læremidler i PDF'en 👀 

Hvis man sender med både overlappsperioder og vedtaksperioder fra sak først, så kan man i endringen av htmlify fjerne overlappsperiode og kun vise vedtaksperiode
Og sen fjerne at man sender overlappsperiode fra sak

Reply:
Hmm 🤔 Tenker egentlig i det er greit å ta det i to steg. I htmlify er det sjekk som gjør at tittelen "Vedtaksperioder" og tillhørende vedtaksperioder kun vises når vedtaksperiodene sendes med. Se `src/internt-vedtak/VedtaksperioderContent.tsx:23`. Kan gjøre det samme for stønadsperioder. Da vil begge kunne leve i koden i en mellomfase også kan vi fjerne stønadsperioder når vi er helt over på vedtaksperioder.

Vedtaksperiodene for læremidler har foreløbig ikke målgruppe og aktivtet. Hvis vi skal sende de til htmlify må vi lage en dto etc. med målgruppe og aktivitet som nullable. Det føles litt unøding når vi uansett snart skal innføre vedtaksperioder med målgruppe og aktivtet også for læremidler. Forslår vi venter til det er på plass.

Reply:
Enig i det!
Hadde ikke sett det med endringen av at stønadsperioder/vedtaksperioder ikke vises hvis listen er tom.

Vil det kunne bli utydelig at tittelen ikke vises i det hele tatt hvis man ikke har noen stønadsperioder? (i de vedtak der man ikke lagt inn noen stønadsperioder og som blir avslag)